README.md

```diff
@@ -6,6 +6,8 @@ This toolset is designed for internal usage and it's in development state. We ar
 
 # Running
 
+We require to use latest [`kubectl`](https://kubernetes.io/docs/tasks/tools/install-kubectl/) version.
```

Comment:
Can we be specific here? Either version greater than X or "we support N last versions of `kubectl`".